specificatie/persoon.yaml

```diff
@@ -158,6 +174,18 @@ components:
           $ref: 'overlijden.yaml#/components/schemas/Overlijden'
         verblijfplaats:
           $ref: 'verblijfplaats.yaml#/components/schemas/AbstractVerblijfplaats'
+        immigratie:
+          $ref: 'immigratie.yaml#/components/schemas/Immigratie'
+        gemeenteVanInschrijving:
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        datumAanvangAdreshouding:
```

Comment:
deze hoort niet in Persoon. datumAanvangAdreshouding en datumAanvangAdresBuitenland worden samengevoegd in datumVan. Die zou ik vervolgens in verblijfplaats verwachten, niet in persoon

Reply:
DatumVan zit in verblijfplaats.yaml